metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/hooks/datahub.py

```diff
@@ -91,7 +92,7 @@ def make_emitter(self) -> "DataHubRestEmitter":
 
         host, token, extra_args = self._get_config_v2()
         return datahub.emitter.rest_emitter.DataHubRestEmitter(
-            host, token, **extra_args
+            fixup_gms_url(host), token, **extra_args
```

Comment:
does it make sense to put this logic in the rest emitter directly?

Reply:
Yeah that is probably the best place. I'll change.